_includes/sections/tor-operating-systems.html

```diff
@@ -0,0 +1,26 @@
+<h1 id="tor_os" class="anchor"><a href="#tor_os"><i class="fas fa-link anchor-icon"></i></a> Tor focused distributions</h1>
+
+<div class="alert alert-warning" role="alert">
+  <strong>These linux distributions are developed with the purpose of directing all network traffic through Tor.</strong>
+</div>
+
+{% include cardv2.html
+  title="Tails"
+  image="/assets/img/svg/3rd-party/tails.svg"
+  description='Tails is a live operating system that can boot on almost any computer from a DVD, USB stick, or SD card you control. It aims at preserving privacy and anonymity, and circumventing censorship by forcing Internet connections through the Tor network; leaving no trace on the computer; and using state-of-the-art cryptographic tools to encrypt files, emails, and instant messages.'
+  badges="info:GNU/Linux"
+  labels="color==warning::text==contrib::tooltip==This software may depend on or recommend non-free software."
+  website="https://tails.boum.org/"
+  git="https://git-tails.immerda.ch/tails/"
+  %}
+
+{% include cardv2.html
+  title="Whonix"
+  image="/assets/img/svg/3rd-party/whonix.svg"
+  description='A Debian-based security-focused Linux distribution. It aims to provide privacy, security and anonymity on the internet. The operating system consists of two virtual machines, a "Workstation"
+    and a Tor "Gateway". All communication are forced through the Tor network to accomplish this.'
```

Comment:
Should we mention...

```suggestion
  description='A Debian-based security-focused Linux distribution. It aims to provide privacy, security and anonymity on the internet. The operating system consists of two virtual machines, a "Workstation" and a Tor "Gateway". All communication are forced through the Tor network to accomplish this. <a href="https://www.whonix.org/wiki/Qubes/Why_use_Qubes_over_other_Virtualizers">Whonix is best used in conjunction with Qubes.</a>'
```

?